source/android/adaptivecards/build.gradle

```diff
@@ -124,11 +120,14 @@ afterEvaluate {
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation 'androidx.gridlayout:gridlayout:1.0.0'
     androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android:flexbox:1.0.0'
     testImplementation 'junit:junit:4.12'
+    implementation "androidx.core:core-ktx:1.5.0-rc02"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
any downsides to this?

Reply:
Known partners (e.g. Teams, Cisco) already use Kotlin (and I suspect most hosts). For them, it should have no impact. For others, there may be a size impact.